shared/storage.ts

```diff
@@ -573,5 +285,20 @@ export async function isMinimumLimitForBackupRestorationReached() {
   // APIs:
   // 1. CloudStorage.stat(file, context)
   // 2. getUncompressedSize()
-  return await Storage.isMinimumLimitReached('minStorageRequired');
+  return await isMinimumLimitReached('minStorageRequired');
```

Comment:
Storage.isMinimumLimitReached was explicitly stating that we are checking storage related limit
But post moving isMinimumLimitReached function outside Storage class this mapping that its related to storage is missing, can we rename function accordingly to state that this function checks storage limit?

Reply:
Agree with @KiruthikaJeyashankar

Reply:
Renamed it to `isMinimumStorageLimitReached`